ex04.ex

```diff
@@ -55,7 +58,14 @@ defmodule Ex04 do
 
   """
 
-  def min . . . "your code"
+
+  def min([ head | tail ]) do
```

Comment:
## -1

This function does a little too much. I'd prefer

~~~ elixir
  def min(list), do: reduce(list, &min_of/2)
  def min_of(a, b) when a < b, do: a
  def min_of(a, b), do: b
~~~